tests/genai/judges/test_judge_tool_get_traces_in_session_tool.py

```diff
@@ -0,0 +1,175 @@
+"""
+Tests for GetTracesInSession.
+
+This module tests the functionality of the GetTracesInSession class,
+including successful trace retrieval, error handling, and parameter validation.
+"""
+
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from mlflow.entities.trace import Trace
+from mlflow.entities.trace_info import TraceInfo as MlflowTraceInfo
+from mlflow.entities.trace_location import TraceLocation
+from mlflow.entities.trace_state import TraceState
+from mlflow.exceptions import MlflowException
+from mlflow.genai.judges.tools.get_traces_in_session import GetTracesInSession
+from mlflow.genai.judges.tools.types import TraceInfo
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE, ErrorCode
+from mlflow.types.llm import ToolDefinition
+
+
+def test_get_traces_in_session_tool_name() -> None:
+    """Test that the tool returns the correct name."""
+    tool = GetTracesInSession()
+    assert tool.name == "get_traces_in_session"
+
+
+def test_get_traces_in_session_tool_get_definition() -> None:
+    """Test that the tool returns a valid tool definition."""
+    tool = GetTracesInSession()
+    definition = tool.get_definition()
+
+    assert isinstance(definition, ToolDefinition)
+    assert definition.function.name == "get_traces_in_session"
+    assert "session" in definition.function.description.lower()
+    assert "multi-turn evaluation" in definition.function.description.lower()
+    assert definition.function.parameters.type == "object"
+    assert len(definition.function.parameters.required) == 0
+    assert definition.type == "function"
+
+    params = definition.function.parameters.properties
+    assert "max_results" in params
+    assert "order_by" in params
+    assert params["max_results"].type == "integer"
+    assert params["order_by"].type == "array"
+
+
+def create_mock_trace(
```

Comment:
might want to remove the trace_id and experiment_id args here since they're uniform at all places of usage.